files/scripts/sonic-netns-exec

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This script will be useful.
How about move it to sonic-utilities, so other could benefit everywhere the package is installed, for example inside a kvm image.

Reply:
Keeping the current location as is, considering similar to other wrappers like redis-cli/vtysh

Reply:
I can understand wanting to move this to sonic-utilities. However, I agree with @SuvarnaMeenakshi -- let's keep it here for now because it is similar to the other wrappers here. If we decide to move them, we should move them all together in a separate PR.